src/app/map/item-location.ts

```diff
@@ -7,13 +7,11 @@ export class ItemLocation {
     private _left: number,
     private _top: number,
     private _availability: () => Availability,
-    private _service: any
+    private _service: any,
+    private _isOpened: boolean = false
```

Comment:
This parameter may end up going away due to later plans. This is not the major focus.